kmesdk/src/main/java/com/kme/kaltura/kmesdk/ws/message/participant/KmeParticipant.kt

```diff
@@ -39,8 +39,7 @@ data class KmeParticipant(
     @SerializedName("long") var long: Double? = null,
 
 //    Local
-    var isSpeaking: Boolean = false,
-    var isHandRaised: Boolean = false
```

Comment:
Removal of this line causes compilation error at multiple part of the project.

Reply:
Done in develop.